cmd/tailscale/cli/whois.go

```diff
@@ -0,0 +1,91 @@
+// Copyright (c) Tailscale Inc & AUTHORS
+// SPDX-License-Identifier: BSD-3-Clause
+
+package cli
+
+import (
+	"context"
+	"encoding/json"
+	"errors"
+	"flag"
+	"fmt"
+	"os"
+	"strings"
+	"text/tabwriter"
+
+	"github.com/peterbourgon/ff/v3/ffcli"
+)
+
+var whoisCmd = &ffcli.Command{
+	Name:       "whois",
+	ShortUsage: "whois [--json] [ip|ip:port]",
```

Comment:
We have --json right after `whois`, but in the PRD examples, it's listed after the IP. Other commands list it at the end of the usage as well (`status [--active] [--web] [--json]`)

Reply:
I tried doing that but somehow the command parser didn't like it. trying again

Reply:
yeah the parser doesn't like it. It just passes the `--json` down in the `args` var

Reply:
Relatedly
 ```
	ShortUsage: "ip [-1] [-4] [-6] [peer hostname or ip address]",
```